corehq/apps/case_importer/views.py

```diff
@@ -70,7 +70,8 @@ def excel_config(request, domain):
         case_upload.check_file()
     except ImporterError as e:
         return render_error(request, domain, get_importer_error_message(e))
-
+    except SpreadsheetFileExtError:
+        return render_error(request, domain, _("Please upload file with extension .xls or .xlsx"))
```

Comment:
do csvs not work here?

Reply:
No @esoergel 

We just 
`ALLOWED_EXTENSIONS = ['xls', 'xlsx']`

https://github.com/dimagi/commcare-hq/blob/5a20d2123e9d68f2b0071aad985e5eccc364641c/corehq/apps/case_importer/views.py#L57